setup.py

```diff
@@ -39,6 +39,8 @@
     'connexion[swagger-ui]>=2.6.0',
     'swagger-ui-bundle>=0.0.6',
     'anyconfig>=0.9.8',
+    'semver>=2.9.1',
```

Comment:
Why is necessary semver? @f-arruza

Reply:
Prance when parsing the OAS, requires semver to check version and doesn't resolve this dependency.